### PR TITLE
Update horizontal-pod-autoscale-walkthrough.md

### PR DESCRIPTION
**Pull request by tengfei86**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by bradtopol** [comment]
@zparnold Are you familiar with this enough to do a technical review? I think you are. If not let me know and I'll find a technical reviewer. Thanks!

/assign @zparnold

**Comment by sftim** [comment]
@tengfei86 I don't see any explanation for what this pull request fixes / improves.

Can you write a note (ideally, in the commit message) to describe why you're making that change? If you're not sure how to do that, reply here and someone else ought to be able to help tweak it.

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 1]
```diff
@@ -304,7 +304,7 @@ spec:
     resource:
       name: cpu
       target:
-        type: AverageUtilization
+        type: Utilization
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md]: According to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#metrictarget-v2beta2-autoscaling,
The type should be one of Utilization, Value, or AverageValue.
So this change makes sense to me.

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 2]
```diff
@@ -322,7 +322,7 @@ spec:
         kind: Ingress
         name: main-route
       target:
-        kind: Value
+        type: Value
```
Comment (tengqm) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md]: According to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#metrictarget-v2beta2-autoscaling,
the target field is a MetricTarget object, which can have fields like averageUtilization, averageValue, type and value. 'kind' is obviously not the right property name.
This change makes sense to me.

**Pull request merged**